src/test/java/com/java/study/group/librarysystem/ITtest/steps/CommonSteps.java

```diff
@@ -0,0 +1,32 @@
+package com.java.study.group.librarysystem.ITtest.steps;
+
+import io.restassured.response.Response;
+import net.thucydides.core.annotations.Step;
+import org.apache.http.HttpStatus;
+
+import static net.serenitybdd.rest.SerenityRest.given;
+import static org.hamcrest.Matchers.is;
+
+public class CommonSteps {
+
+    public Response response;
+
+    @Step("Given a GET request to the webapp for the path {0}")
+    public void makeGetRequestToApplication(final String path, String... args) {
+        response = given()
+                .when()
+                .get(path, (Object[]) args);
+    }
+    @Step("A {0} response code should be provided")
+    public void verifyResponseStatusCode(int statusCode) {
```

Comment:
In case you cannot reuse this method in other tests, you can remove this since is duplicated `verifyApplicationIsUp()`